src/handlers/edit.ts

```diff
@@ -0,0 +1,42 @@
+import { Task } from "../models/Task";
+import { getTasks, saveTasks } from "../services/tasks";
+
+export namespace EditHandler {
+  export type Args = {
+    id: string;
+    done: boolean;
+    pending: boolean;
+    description?: string;
+    by?: string;
+  };
+
+  export const handle = async (args: Args): Promise<void> => {
+    let allTasks: Array<Task> = await getTasks();
+
+    let task: Task | undefined = allTasks.find((task) => task.id === args.id);
```

Comment:
(nit) Both of these variables could be `const`

Reply:
That's true! I'll fix it